src/mod_bosh_socket.erl

```diff
@@ -192,11 +205,17 @@ get_cached_responses(Pid) ->
 %%--------------------------------------------------------------------
 init([HostType, Sid, Peer, PeerCert]) ->
     BoshSocket = #bosh_socket{sid = Sid, pid = self(), peer = Peer, peercert = PeerCert},
-    C2SOpts = #{access => all, shaper => none, xml_socket => true, hibernate_after => 0},
-    {ok, C2SPid} = ejabberd_c2s:start({mod_bosh_socket, BoshSocket}, C2SOpts),
+    C2SOpts = #{access => all,
+                shaper => none,
+                xml_socket => true,
+                max_stanza_size => 0,
+                hibernate_after => 0,
+                c2s_state_timeout => 5000,
+                backwards_compatible_session => true},
```

Comment:
Huh, we should have these from the config, but anyway, nobody uses bosh, nor they should anyway 🤷🏽